packages/cli/src/commands/list/index.ts

```diff
@@ -23,6 +24,18 @@ import { listCommand } from './command';
 import parseTarget from '../../util/parse-target';
 import { getFlagsSpecification } from '../../util/get-flags-specification';
 
+function ToDate(timestamp: number): string {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** camelCase

```suggestion
function toDate(timestamp: number): string {
```